api/analyzedphenotypes.datatype.api.inc

```diff
@@ -0,0 +1,286 @@
+<?php
+/**
+ * @file
+ *
+ * API: UNIT AND DATA TYPES
+ * Manage property and operation pertaining to trait unit and data type.
+ */
+
+
+/**
+ * DATA SOURCE
+ * Define data type, name and symbol of a unit.
+ *
+ * @return array.
+ *   An array containing the name and the data type of a unit. Data types:
+ *   - alphanumeric    : Letters, number combination.
+ *   - trait           : String value strickly in - Trait name (unit) additional text - format.
+ *   - text            : Letters only.
+ *   - number          : Numbers + 0.
+ *   - number_no_zero  : Numbers > 0.
+ *   - scale_1-5       : 1,2,3,4 and 5.
+ *   - scale_1-10      : 1 - 10.
+ *   - y/n/?           : y, yes, n, no, unsure, ?.
+ *   - date            : YYYY-MM-DD.
+ *   - year            : YYYY
+ */
+function ap_define_datatypes() {
+  $datatype = array(
+    'g' => array('name' => 'grams',    'type' => 'number'),
+    'kg' => array('name' => 'kilograms',  'type' => 'number'),
+    'cm'  => array('name' => 'centimeters', 'type' => 'number'),
+    'mg'   => array('name' => 'milligrams',   'type' => 'number'),
+    'mM'    => array('name' => 'milimolar',  'type' => 'number'),
+    'days'   => array('name' => 'days',     'type' => 'number_no_zero'),
+    'days'    => array('name' => 'days',   'type' => 'number_no_zero'),
+    'days'     => array('name' => 'days', 'type' => 'number_no_zero'),
```

Comment:
Why is 'days' repeated 3 times?

Reply:
Good catch  👍  ! I meant one to be date and the remaining days to go away. Not sure what you mean with the very minor suggestion :)